src/common/containers/build-details.js

```diff
@@ -80,10 +78,15 @@ class BuildDetails extends Component {
             </div>
           </div>
         }
-        <HelpInstallSnap
-          headline='To test the latest successful build on your PC or cloud instance:'
-          name='foo'
-        />
+        { isFetching &&
+          <div className={styles.spinner}><Spinner /></div>
+        }
+        { snap &&
+          <HelpInstallSnap
+            headline='To test the latest successful build on your PC or cloud instance:'
+            name={ snap.store_name }
```

Comment:
bike shed: `store_name` requires you to know what the store is in this case? so might be better as `registered_name`